paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h

```diff
@@ -31,6 +33,44 @@ using LoDTensor = framework::LoDTensor;
 using SelectedRows = framework::SelectedRows;
 using DDim = framework::DDim;
 
+#if defined(PADDLE_WITH_MKLML) && !defined(_WIN32) && !defined(__APPLE__) && \
+    !defined(__OSX__) && !defined(PADDLE_WITH_CUDA)
+template <typename T>
+void prepare_csr_data(const std::vector<uint64_t> &offset,
+                      const int64_t *ids_data, const size_t idx_width,
+                      T *csr_vals, int *csr_colmuns, int *csr_row_idx) {
+  int val_idx = 0;
+  int row_idx = 0;
+  csr_row_idx[0] = 0;
+
+  std::map<int, int> ids_map;
+
+  // for each sequence in batch
```

Comment:
which var indicate batch? offset or ids_data? a bit confusing here.

Reply:
The batch information is included in offset vector.